projects/uitdatabank/docs/errors.md

```diff
@@ -125,3 +125,21 @@ Subsequently, appropriate actions, such as updates to an existing Place, can be
   "duplicatePlaceUrl": "/place/581314d4-637e-407b-ba35-8b60847012d0"
 }
 ```
+
+## failed-subscription-to-newsletter
```

Comment:
This error is public?

Reply:
Define public? It is throw by an internally marked API, but technically it is available publicly.
Either way I was not sure if I had to document the error or not, but if I remove this the URL in the UDB3 code will not point to anything.
Happy to go with whatever the team wants. :-)

Reply:
As discussed on Slack, best to remove the error from the markdown file because this file is always public. So it could become confusing for a developer to see this error. Furthermore, the RDF errors are also not documented.